### PR TITLE
refactor(frontend): migrate add blockchain account form

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/components/accounts/blockchain/AddressInput.vue [hunk 1]
```diff
@@ -86,56 +86,53 @@ const rules = [
...
+    </RuiCheckbox>
+    <VTextField
+      v-if="!multiple"
+      v-model="address"
+      data-cy="account-address-field"
+      outlined
+      class="account-form__address"
+      :label="t('common.account')"
+      :rules="rules"
+      :error-messages="errorMessages"
```
Comment (kelsos) [inline frontend/app/src/components/accounts/blockchain/AddressInput.vue:108]: Any reason to not migrate it to vuelidate and RuiTextField?

File: frontend/app/src/components/accounts/management/inputs/AllEvmChainsSelector.vue [hunk 2]
```diff
@@ -17,15 +17,23 @@ const { t } = useI18n();
...
-      :input-value="value"
-      :disabled="disabled"
-      class="py-4 px-4 my-0"
-      :label="t('address_account_form.label')"
-      persistent-hint
-      :hint="t('address_account_form.hint')"
-      @change="emit('input', !value)"
-    />
-  </VSheet>
+  <RuiCard outlined class="rounded">
```
Comment (kelsos) [inline frontend/app/src/components/accounts/management/inputs/AllEvmChainsSelector.vue:20]: outlined should be no-op here right?

**Comment by lukicenturi** [comment]
> Now that I merged my PR we might be able to also migrate VBtnToggle

Need this to be merged first https://github.com/rotki/ui-library-compat/pull/40

File: frontend/app/src/components/accounts/blockchain/AddressInput.vue [hunk 3]
```diff
@@ -79,63 +83,96 @@ const setAddress = (addresses: string[]) => {
...
 onMounted(() => setAddress(get(addresses)));
 
-const rules = [
-  (v: string) =>
-    !!v || t('account_form.validation.address_non_empty').toString()
-];
+const rules = {
+  address: {
+    required: helpers.withMessage(
+      t('account_form.validation.address_non_empty').toString(),
```
Comment (kelsos) [inline frontend/app/src/components/accounts/blockchain/AddressInput.vue:89]: for a followup toString() is now redundant so we can remove it.

File: frontend/app/src/components/accounts/blockchain/XpubInput.vue [hunk 4]
```diff
@@ -1,7 +1,6 @@
 \
```
Comment (kelsos) [inline frontend/app/src/components/accounts/blockchain/XpubInput.vue:1]: also for the follow up this :)

**Review by kelsos — APPROVED** [review]
lgtm, left two things for a followup

**Pull request merged**